src/bin/club-create

```diff
@@ -30,70 +22,34 @@ program
     .option('-s, --state [id|name]', 'Set workflow state of story', '')
     .option('-y, --type [name]', 'Set type of story, default: feature', 'feature')
     .parse(process.argv);
-spin.setSpinnerString(27);
 
 const main = async () => {
+    const entities = await storyLib.fetchEntities();
     if (!program.idonly)
         spin.start();
-    try {
-        [ wfs, members, projects, epics ] = await Promise.all([
-            client.listWorkflows(),
-            client.listMembers(),
-            client.listProjects(),
-            client.listEpics(),
-        ]);
-        wf = wfs[0];    // TODO: this is always getting the default workflow
-    } catch (e) {
-        log('Error fetching workflows');
-    }
     let update = {
         name:          program.title,
         story_type:    program.type,
         description:   program.description,
         estimate:      program.estimate || undefined
     };
     if (program.project) {
-        const projectMatch = new RegExp(program.project, 'i');
-        const project = projects.filter(s => {
-            return !!`${s.id} ${s.name}`.match(projectMatch);
-        })[0];
-        update.project_id = (project || {}).id;
+        update.project_id = (storyLib.findProject(entities, program.project) || {}).id;
     }
     if (program.state) {
-        const stateMatch = new RegExp(program.state, 'i');
-        const newState = wf.states.filter(s => {
-            return !!`${s.id} ${s.name}`.match(stateMatch);
-        })[0];
-        update.workflow_state_id = (newState || {}).id;
+        update.workflow_state_id = (storyLib.findState(entities) || {}).id;
```

Comment:
Is there a reason not to pass `program.state` here?

Reply:
Typo, that's the reason.